app/assets/javascripts/admin/views/project/project_list_view.js

```diff
@@ -82,11 +83,19 @@ class ProjectListView extends React.PureComponent<StateProps, State> {
     });
   };
 
+  _setState = (...args) => this.setState(...args);
```

Comment:
This is repeated in every component using the `StatePersistenceComponent`. Perhaps we can find a clever way to include this into it directly.